mchf-eclipse/drivers/audio/audio_driver.c

```diff
@@ -4670,10 +4696,14 @@ static void AudioDriver_TxProcessor(AudioSample_t * const srcCodec, IqSample_t *
 
         if (signal_active)
         {
+    #if defined(UI_BRD_OVI40)
+            // FIXME -> there is hard coded gain, should be defined or be adjustable.
+            AudioDriver_FillSideToneAudioBuffer( adb.i_buffer, audioDst, blockSize, 0.1 );
+    #endif
```

Comment:
I am not sure this works as expected. CW does not have signal active all the time and then the audio buffer contains whatever was there last time. In the original place we mute the signal in dst AND then copy it to audio, so there is no problem.

Reply:
I see what do you mean, so far working fine, clean and nice sidetone. To be save changed a bit, take a look...